indexer/api/handlers/deposits.go

```diff
@@ -0,0 +1,94 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/ethereum-optimism/optimism/indexer/api/middleware"
+	"github.com/ethereum-optimism/optimism/indexer/database"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/go-chi/chi/v5"
+)
+
+type DepositItem struct {
+	Guid string `json:"guid"`
+	From string `json:"from"`
+	To   string `json:"to"`
+	// TODO could consider OriginTx to be more generic to handling L2 to L2 deposits
+	// this seems more clear today though
+	Tx      Transaction `json:"Block"`
```

Comment:
This changes in future prs. Most of this is just kinda here as a starting point not meant to be the final landing place